src/components/ContextMenu/index.js

```diff
@@ -90,7 +90,7 @@ class ContextMenu {
                     var childMenuItemElement = new Container({ class: CLASS_ContextMenu_child });
                     childMenuItemElement.dom.setAttribute("style", `top: ${j * 27.0}px; left: 150px;`);
                     childMenuItemElement.on('click', (e) => {
-                        e.stopPropagation(); removeMenu(); childItem.onClick();
+                        e.stopPropagation(); removeMenu(); childItem.onClick(e);
```

Comment:
```suggestion
                            e.stopPropagation();
                            removeMenu();
                            childItem.onClick(e);
```

Reply:
I realize this style of multiple lines of logic on the same line is used elsewhere...but just wanted to flag that this can make debugging (and readability) a pain. Feel free to ignore this suggestion - up to you.

Reply:
Thanks! I decided to be brave and fix both occurrences of the strange layout. :)